fire/SFMainMod.F90

```diff
@@ -916,21 +922,53 @@ subroutine  crown_damage ( currentSite )
 
           do while(associated(currentCohort))  
              currentCohort%fraction_crown_burned = 0.0_r8
+             currentCohort%crown_FI              = 0.0_r8  !critical fire intensity for passive crown fire
+             currentCohort%crown_fire_flg        = 0       !flag for passvie crown fire ignition
+             
              if (EDPftvarcon_inst%woody(currentCohort%pft) == 1) then !trees only
+                
+                ! Calculate clear branch bole height at base of crown
+                ! inst%crown = crown_depth_frac (PFT)
+                height_cbb = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)
+                
+                ! Evaluate for passive crown fire ignition
+                if (EDPftvarcon_inst%crown_fire(currentCohort%pft) == 1) then
+                   
+                   ! Crown fuel ignition potential, EQ 8 Bessie and Johnson 1995
+                   currentCohort%crown_FI = (0.01 * height_cbb * SF_val_crown_ignition_energy)**1.5
+
+                   if (currentCohort%crown_FI >= crown_fire_threshold) then ! 200 kW/m = threshold for crown fire potential
+                      
+                      ! Initiation of passive crown fire, EQ 9 Bessie and Johnson 1995                     
+                      currentCohort%ignite_crown = currentPatch%FI/currentCohort%crown_FI
+                      if (currentCohort%ignite_crown > 1.0_r8) then
+                         currentCohort%crown_fire_flg = 1  ! passive crown fire ignited
+                         currentCohort%fraction_crown_burned =  1.0_r8
+                      else ! evaluate crown damage based on scorch height
```

Comment:
```suggestion

```

Reply:
...all the way to just before end if (line 957)

Reply:
@slevisconsulting I made these updates with last commit